test_driver/auth.dart

```diff
@@ -26,6 +27,7 @@ void main() {
 
   // create a fake redux bundle
   final redux = ReduxBundle(
+      notificationsService: FakeNotificationsService(),
```

Comment:
added an extra fake service to avoid the prompt from the `firebase_messaging` plugin that we can't interact with via flutter_driver

Reply:
noice